cirq-core/cirq/transformers/gauge_compiling/sqrt_cz_gauge.py

```diff
@@ -14,16 +14,44 @@
 
 """A Gauge transformer for CZ**0.5 gate."""
 
+import numpy as np
+
 from cirq.transformers.gauge_compiling.gauge_compiling import (
     GaugeTransformer,
     GaugeSelector,
     ConstantGauge,
+    Gauge,
 )
 from cirq.ops.common_gates import CZ
 from cirq import ops
```

Comment:
Nit - we reference the CZ gate directly, but refer to X and S as ops.X, ops.S.
For the sake of consistency, can we use `ops.CZ` instead of CZ?

Reply:
done